test/testcafe/spec/DeviceChallengePollView_spec.js

```diff
@@ -486,3 +495,21 @@ test
     )).eql(1);
     await t.expect(loopbackOVFallbackLogger.contains(record => record.request.url.match(/6513/))).eql(false);
   });
+
+test
+  .requestHooks(customURILoginHintMock)('pass login_hint to Okta Verify through Custom URI', async t => {
+    const identityPage = await setupLoopbackFallback(t);
+    await renderWidget({
+      features: { engFastpassMultipleAccounts: true },
+    });
+
+    const username = 'john.smith@okta.com';
+    await identityPage.fillIdentifierField(username);
+    identityPage.clickOktaVerifyButton();
+    const deviceChallengePollPageObject = new DeviceChallengePollPageObject(t);
+    await t.expect(deviceChallengePollPageObject.getHeader()).eql('Click "Open Okta Verify" on the browser prompt');
+
+    // verify login_hint has been appended to the customURI url in the iframe
+    const attributes = await Selector('#custom-uri-container').attributes;
```

Comment:
put this `Selector` into a page object for eg in `DeviceChallengePollPageObject`